daemon/container.go

```diff
@@ -73,6 +73,7 @@ type CommonContainer struct {
 	MountLabel, ProcessLabel string
 	RestartCount             int
 	UpdateDns                bool
+	HasBeenStartedBefore     bool
```

Comment:
It allows us to do a first boot optimization on Windows when starting the container.